src/test/java/seedu/address/logic/commands/DeleteModuleCommandTest.java

```diff
@@ -37,13 +34,6 @@ public void execute_validNameUnfilteredList_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
-    @Test
```

Comment:
why is this test deleted?

Reply:
Now checking for invalid module name is done under DeleteModuleCommandParser, so we don't have to test it here since DeleteModuleCommand will never be called with an invalid ModuleName. Same for other test files as well

Reply:
Thanks for the explanation!